starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ItemsObject.java

```diff
@@ -0,0 +1,49 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by jay on 10/25/16. */
+
+public class ItemsObject {
+    String mItemName, mDescription, mType;
+    String mPrice;
+
+    public ItemsObject(String mItemName, String mDescription, String mType, String mPrice) {
+
+        this.mItemName = mItemName;
+        this.mDescription = mDescription;
+        this.mType = mType;
+        this.mPrice = mPrice;
+    }
+
+    public String getmItemName() {
+        return mItemName;
+    }
+
+    public void setmItemName(String mItemName) {
+        this.mItemName = mItemName;
```

Comment:
Totally fine to have setters here, but I just want to make sure you know you don't always _need_ to have setters. Sometimes you don't want an object's values to change after it's created, so you might prefer not to provide any setters.